EIPS/eip-8261.md

```diff
@@ -0,0 +1,94 @@
+---
+eip: 8261
```

Comment:
```suggestion
eip: 7637
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500.

Please also update the filename.